src/components/Pages/PlaylistCollection/Page.js

```diff
@@ -65,9 +65,11 @@ const PlaylistCollectionPage = ({ collection, nextLink = null, prevLink = null,
 
   const handleSelectedChange = useCallback(nSelected => {
     if (nSelected !== selected && playlist?.items && playlist?.items[nSelected]) {
-      history.push(`/${uiLanguage}${playlist.items[nSelected].shareUrl}`);
+      // console.log('share url:', playlist.items[nSelected].shareUrl)
+      // history.push(`/${uiLanguage}${playlist.items[nSelected].shareUrl}`);
```

Comment:
@wissog why you remove it? is bug that it was fixe not reproduce? cant find how you fixe it

Reply:
this causes to open the next item in a new page instead of remaining in the playlist